### DISCUSSION TITLE
V2.12.0 Shareinto Question Result Changed From V2.11.0

### DISCUSSION THREAD

**Discussion by RalphHightower** [gsantner/markor#2281] [General] [answered]
When I want to save a link to a web page, using the Chrome browser, I would: 
1. click on the 3 horizontal dots from the browser;
2. select share;
3. choose Markor and output to QuickLinks.

I use this to keep a portable list of bookmarks. The output from V2.11.0 conveys the exact information that I want to include in my bookmarks or for use on my website. 

I've looked for solutions and I have found nothing. I've experimented with the Shareinto settings with no success. For the Markor output, I backslashed the opening bracket to show what is put into QuickLinks. Markor has been great to use.

V2.12.0:
\[scitechdaily](https://scitechdaily.com/the-craton-enigma-scientists-propose-a-new-continental-formation-theory/)

V2.11.0:
\[The Craton Enigma: Scientists Propose a New Continental Formation Theory](https://scitechdaily.com/the-craton-enigma-scientists-propose-a-new-continental-formation-theory/ )

**Comment by harshad1** [comment]
I see what happened here. Will take a look and fix it if possible.

**Comment by harshad1** [comment]
FYI, this should be an issue :)

**Reply by RalphHightower** [reply]
Thank you. I didn't know if this was a planned change or not.

**Comment by harshad1** [comment, marked as answer]
I knocked a fix out - #2282 

If it works (lots of testing needed), it will hopefully be available in the next release or 2

**Reply by RalphHightower** [reply]
I have a workaround for V2.12.0:
1. Copy the headline text to the clipboard.
2. Shareinto for the web page.
3. Insert the headline in front of the link.
4. Insert the appropriate brackets and parentheses. 
5. Move it to QuickLinks.

**Reply by harshad1** [reply]
Yeah. That works temporarily until my fix is in

**Reply by tengucrow** [reply]
Didn't see this discussion so I opened an issue "sharing link from browser lost ist header" #2294 

If the bug is already known, you can close it.

**Reply by gsantner** [reply]
Further discuss at #2294 , might be eaiser to discover.
